src/screens/Gurbani/Line.tsx

```diff
@@ -5,26 +5,20 @@ import Typography from '../../components/Typography'
 import Languages from '../../lib/languages'
 import transliterators from '../../lib/transliterators'
 import Colors from '../../themes/colors'
-import Fonts from '../../themes/fonts'
 import { px, py } from '../../themes/utils'
 import { TranslationData } from '../../types/data'
 
 const styles = StyleSheet.create( {
   gurbani: {
-    color: Colors.PrimaryText,
-    fontFamily: Fonts.OpenGurbaniAkharBlack,
-    fontSize: 16,
-    lineHeight: 24,
+    fontSize: 17 * ( 16 / 14 ),
   },
   root: {
-    ...px( 12 ),
-    ...py( 12 ),
+    ...px( 20 ),
+    ...py( ( 17 * 1.75 ) / 2 ),
   },
   text: {
     color: Colors.SecondaryText,
-    fontSize: 14,
-    lineHeight: 24,
-    paddingTop: 6,
+    paddingTop: ( 17 * 1.75 ) / 4,
```

Comment:
You may want to give these magic numbers a variable declaration above, or if they're global, export them in our styles

Reply:
Which style file should I globalize them in? Typography?

Reply:
maybe something in the `themes` folder - perhaps a new file called units or typography